dataminer/Administrator_guide/Failover/Configuring_Failover/Preparing_the_two_DataMiner_Agents.md

```diff
@@ -27,6 +27,8 @@ In addition, make sure the [required ports are opened](#opening-the-required-por
 
 The primary DMA is a normal DataMiner Agent. In most cases, this will be an existing DMA that is a member of a DMS cluster. It does not require any additional configuration.
 
+For a brand-new Failover pair where the primary DMA is not yet part of a cluster, the correct procedure is to first add the primary DMA to the cluster using its physical IP. Only after that step should you set up the Failover by using the Failover button and adding the backup (empty) DMA.
+
 ## Backup DataMiner Agent
 
 The backup DMA must be a newly installed DataMiner Agent.
```

Comment:
```suggestion
The backup DMA must be a newly installed DataMiner Agent.

If you are adding a brand-new Failover pair to a cluster, you do not need to add the backup DMA to the cluster yourself. Once the Failover setup is complete, it will be added automatically.
```